hexrdgui/hexrd_config.py

```diff
@@ -888,13 +888,13 @@ def images_dict(self):
 
     @property
     def raw_masks_dict(self):
-        return self.create_raw_masks_dict(display=False)
+        return self.create_raw_masks_dict(self.images_dict, display=False)
 
-    def create_raw_masks_dict(self, display=False):
+    def create_raw_masks_dict(self, images_dict, display=False):
         """Get a masks dict"""
         from hexrdgui.masking.mask_manager import MaskManager
         masks_dict = {}
-        for name, img in self.images_dict.items():
```

Comment:
We are avoiding calling `self.images_dict` here because it is a computed property and takes some time to generate. Instead, the already-created `images_dict` is passed in as a function argument.